site/how-it-works.md

```diff
@@ -0,0 +1,86 @@
+---
+title: How Weave Flux Works
+menu_order: 70
+---
+
+This page describes the goals of flux, how it achieves them and 
+significant architectural decisions. It is intentionally high level 
+to prevent it from being out of date too quickly.
+
+# Goals
+
+The overall goal of Flux is to automate the deployment of services.
+A typical use case would be:
+
+1. A developer merges their tested code into a branch which creates an
+   artifact as a (Docker) container
+2. Flux detects the presence of a new container and deploys that to an
+   orchestrator (e.g. Kubernetes)
+3. Flux writes the new cluster configuration to version control
+4. Changes are made visible through chat integration and audit logging
+ 
+Hence, the goal is to automate away the need for a developer to 
+interact with an orchestrator (which is a common source of accidental
+failure) or with the systems that ensure that the orchestrator is in
+a working state.
+
+# Implementation Overview
```

Comment:
Yes, it appears that it is: e.g. https://www.weave.works/docs/net/latest/introducing-weave/